### PR TITLE
[WEB-706] react upgrade

### PR DESCRIPTION
**Pull request by krystophv**
[WEB-706] - upgrading react to latest version

accompanied by:
https://github.com/tidepool-org/viz/pull/186
https://github.com/tidepool-org/blip/pull/650

[WEB-706]: https://tidepool.atlassian.net/browse/WEB-706

### CODE REVIEW

File: example/components/modalsubnav.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+var PropTypes = require('prop-types');
 /*
```
Comment (cbwebdevelopment) [inline example/components/modalsubnav.js]: Suggest removing licence here

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,11463 @@
+{
```
Comment (cbwebdevelopment) [inline package-lock.json]: Did you mean to commit a package-lock.json?

Comment (krystophv) [reply package-lock.json]: Nah, that one slipped through

File: example/components/modalsubnav.js (hunk 1 above)
Comment (krystophv) [reply example/components/modalsubnav.js]: Suggestion accepted :)

**Pull request merged**